tests/security/usbguard/usbguard.pm

```diff
@@ -0,0 +1,138 @@
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# Summary: Test 'usbguard' can work:
+#          '# systemctl enable usbguard.service',
+#          '# systemctl start usbguard.service',
+#          '# systemctl status/is-active usbguard.service',
+#          '# usbguard generate-policy', '# usbguard allow-device',
+#          '# usbguard list-devices', '# usbguard block-device'
+#          '# check/cat '/var/log/usbguard/usbguard-audit.log',
+#          '# usbguard add-user', '# usbguard remove-user',
+#          '# usbguard get-parameter', '# usbguard set-parameter'
+#          '# usbguard generate/list/install/remove rules'
```

Comment:
would you like to provide a Jira ticket id & subject on the summary or somewhere as it is a new feature case?

Reply:
I have invoked jira ticked in poo / tc

Reply:
I can also search some information via Jira in the project. But I think it's fine if we have a concern about it, we can refer to poo instead. But sometimes we cannot see the detail in poo.

Reply:
I can also see your git commit message here. It's fine for me.
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/13695/commits